### ISSUE TITLE
MACE framework，library "libcdsprpc.so" not found

### ISSUE THREAD

**Issue by hzwangjialei** [XiaoMi/mobile-ai-bench#4]
`python tools/benchmark.py --output_dir=output --frameworks=MACE --runtimes=CPU --model_names=MobileNetV1 --target_abis=armeabi-v7a`
when run the cmd above， it errors on MACE，   NCNN and SNPE framework are able to run successfully
`('## make_output_processor ', <function process_output at 0x1042a4848>)
('cmd :', "cd /data/local/tmp/aibench; ADSP_LIBRARY_PATH='.;/system/lib/rfsa/adsp;/system/vendor/lib/rfsa/adsp;/dsp'; LD_LIBRARY_PATH=. ./model_benchmark")
('frameworks :', ['MACE'])
('MACE', 'CPU', 'MobileNetV1')
WARNING: linker: ./model_benchmark: unused DT entry: type 0xf arg 0xa6a
CANNOT LINK EXECUTABLE: library "libcdsprpc.so" not found
page record for 0xf769503c was not found (block_size=16)`

**Comment by hzwangjialei** [comment]
solved. some phone without dsp,  so can't  find libcdsprpc.so.   it runs successfuly on XiaoMi with SnapDragon 835

**Comment by lee-bin** [comment]
Yes, you are right. Fixed in c141657f01fc36b5e8474d60e2647b3153584740. Now mace can run on CPU/GPU without `libcdsprpc.so`.

**Comment by cvJonathan** [comment]
it works!  thanks！  Incidentally，could you tell me  the version of Mace（libmace.zip）?
